### PR TITLE
Front-end workshop based.

### PR DESCRIPTION
**Pull request by prerit2010**

### CODE REVIEW

**Comment by prerit2010** [comment]
Hi @rgaiacs , @pbanaszkiewicz 
please look at the data we can retrieve on selecting a particular failed package. Data is about system environments which were used while trying to check for installation of that package through installation scripts. For example, for pandas :  http://installation.software-carpentry.org/view/detail/?package_detail=Pandas+%28pandas%29%7C1.1

The numbers corresponding to every detail show the number of users who used that version etc, e.g : **"Ubuntu" : 4** shows that 4 users used Ubuntu.

What type of charts can we think of using this data ?

**Comment by pbanaszkiewicz** [comment]
@prerit2010 the simplest seems to be: bar-charts for each entry in the `user_system_info`.

**Comment by prerit2010** [comment]
That's what I could think off. :+1:

File: app/templates/index.html [hunk 1]
```diff
@@ -49,31 +72,56 @@ <h3 class="panel-title">Most Failed Packages</h3>
...
+				  					</select>
+									<div class="checkbox">
+						  				{% if not show_all%}																				  			  		<label><input type="checkbox" name="all_workshops" value="True">All workshops</label>
+								  		{% else %}
+								  	  		<input type="hidden" value="True" name="all_workshops">
+									  	{% endif %}
+				  					</div>
+									{% if not show_all %}
+						  				<input type="hidden" value="{{workshop_name}}" name="workshop_name">
+						  			{% endif %}
```
Comment (pbanaszkiewicz) [inline app/templates/index.html]: This is not rule, just a suggestion, but you should consider maybe having 2 spaces indents in HTML/Django template files, and 4 spaces (as PEP8 suggests) in Python files.

Tab indents are always displayed differently depending on the tool, which is a bad thing. Here for example GitHub uses something like 8-spaces indents to display tab indents, but my editor uses 4 spaces to display them.

Comment (prerit2010) [inline app/templates/index.html]: Yes yes, I do convert tab to spaces. Just forgot to do it here as well. Will do it.

File: app/views.py [hunk 2]
```diff
@@ -101,37 +103,135 @@ def installation_data():
...
+    user_info = db.session.query(UserSystemInfo.workshop_id.distinct().label("workshop_id")).all()
+    workshops = [
+        user.workshop_id for user in user_info 
+        if user.workshop_id is not None
+    ]
+
+    return render_template('index.html', response=response, workshops=workshops, show_all=True)
+
+
+@application.route('/view/<workshop_id>/')
```
Comment (rgaiacs) [inline app/views.py]: Can we provide the list of workshops on the database?

**Pull request merged**